views/admin/inc/live_chat.tpl

```diff
@@ -0,0 +1,77 @@
+<style>
+  .livezilla-wrap {
+    border-spacing: 0;
+    color: rgb(99, 56, 54);
+    font-size: 14px;
+    letter-spacing: normal;
+    line-height: 1.3;
+    border: 0;
+    vertical-align: middle;
+    background: initial;
+    min-height: 0;
+    padding: 0;
+    font-family: 'Roboto',arial,Helvetica,verdana,sans-serif;
+    -webkit-font-smoothing: antialiased;
+    text-align: left;
+    transform: none;
+    box-sizing: initial;
+    display: block;
+    position: fixed;
+    width: 48px;
+    height: 48px;
+    margin: 0px 30px 46px 0px;
+    left: auto;
+    top: auto;
+    right: 0px;
+    bottom: 0px;
+    z-index: 20000000;
+  }
+
+  .livezilla-wrap .livezilla-btn {
+    user-select: none;
+    box-sizing: content-box;
+    margin: 0;
+    font-size: 14px;
+    letter-spacing: normal;
+    line-height: normal;
+    border: 0;
+    vertical-align: middle;
+    min-height: 0;
+    padding: 0;
+    cursor: pointer;
+    box-shadow: 0 5px 36px rgba(0,0,0,.18);
+    background: rgb(255, 32, 20);
+    height: 48px;
+    width: 48px;
+    z-index: 20000200;
+    border-color: rgb(240, 12, 0);
+    transform: translateY(0px);
+    right: 0px;
+    border-radius: 50%;
+    position: relative;
+    display: block;
+    top: 0px;
+    visibility: visible;
+  }
+
+  .livezilla-wrap .livezilla-btn svg {
+    fill: #fff;
+    position: relative;
+    z-index: 20000299;
+    width: 48px !important;
+    height: 48px !important;
+  }
+
+  .livezilla-wrap .livezilla-btn:hover {
+    box-shadow: 0 5px 36px rgba(0,0,0,.30);
+    transform: rotate(4deg);
+  }
+</style>
+
+<div class="livezilla-wrap">
+  <a class="livezilla-btn" href="javascript:void(window.open('http://www.provusgroup.com/livezilla/chat.php?hfk=MQ__','','width=400,height=600,left=0,top=0,resizable=yes,menubar=no,location=no,status=yes,scrollbars=yes'))">
+    <svg width="48" height="48" style="width:48px !important;height:48px !important" viewBox="-12 -11 48 48" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Probably not relevant, but this won't work in IE < 10.

Reply:
The code for the "frontend" button was taken from the HTML generated by the provided javascript.